.travis.yml

```diff
@@ -0,0 +1,8 @@
+sudo: false
+language: node_js
+node_js:
+  - "node"
+  - "4.8"
```

Comment:
We should document the minimal Node version in the README and migration guide. We can also add it to the package.json [engines](https://docs.npmjs.com/files/package.json#engines)